--- adminutil-1.1.6/Makefile.am.orig	2007-10-11 04:32:43.000000000 +0200
+++ adminutil-1.1.6/Makefile.am	2008-07-21 22:04:32.156744273 +0200
@@ -19,7 +19,8 @@
 #------------------------
 NSPR_LINK = @nspr_lib@ -lplc4 -lnspr4
 NSS_LINK = @nss_lib@ -lssl3 -lnss3
-LDAPSDK_LINK = @ldapsdk_lib@ -lssldap60 -lldap60 -lprldap60 -lldif60
+LDAPSDK_LINK = @ldapsdk_lib@ -lssldap60 -lldap60 -lprldap60
+LDAPSDK_NOSSL_LINK = @ldapsdk_lib@ -lldap60 -lprldap60
 if enable_tests
 SASL_LINK = @sasl_lib@ -lsasl2
 endif
@@ -44,7 +45,7 @@
 TESTS = retrieveSIE retrieveSIEssl psetread psetreadssl psetwrite
 
 TEST_CPPFLAGS = $(AM_CPPFLAGS) -I$(srcdir)/include @nss_inc@ @nspr_inc@ @ldapsdk_inc@ @icu_inc@
-TEST_LDADD = libadmsslutil.la libadminutil.la $(NSPR_LINK) $(NSS_LINK) $(LDAPSDK_LINK) $(SASL_LINK) $(ICU_LINK)
+TEST_LDADD = libadmsslutil.la libadminutil.la $(NSPR_LINK) $(NSS_LINK) $(LDAPSDK_LINK) $(ICU_LINK)
 
 retrieveSIE_SOURCES = tests/retrieveSIE.c
 retrieveSIE_CPPFLAGS = $(TEST_CPPFLAGS)
@@ -123,7 +124,7 @@
 	lib/libadminutil/uginfo.c
 
 libadminutil_la_CPPFLAGS =  $(AM_CPPFLAGS) @nspr_inc@ @ldapsdk_inc@ @icu_inc@
-libadminutil_la_LIBADD =  $(NSPR_LINK) $(NSS_LINK) $(LDAPSDK_LINK) $(SASL_LINK) $(ICU_LINK)
+libadminutil_la_LIBADD =  $(NSPR_LINK) $(LDAPSDK_NOSSL_LINK) $(ICU_LINK)
 libadminutil_la_LDFLAGS = -version-number $(subst .,:,$(PACKAGE_VERSION))
 if CXXLINK_REQUIRED
 libadminutil_la_LINK = $(CXXLINK)
@@ -143,7 +144,7 @@
 	lib/libadmsslutil/uginfossl.c
 
 libadmsslutil_la_CPPFLAGS = $(AM_CPPFLAGS) @nss_inc@ @nspr_inc@ @ldapsdk_inc@ @icu_inc@
-libadmsslutil_la_LIBADD = $(NSPR_LINK) $(NSS_LINK) $(LDAPSDK_LINK) $(SASL_LINK) $(ICU_LINK) libadminutil.la 
+libadmsslutil_la_LIBADD = $(NSPR_LINK) $(NSS_LINK) $(LDAPSDK_LINK) $(ICU_LINK) libadminutil.la
 libadmsslutil_la_LDFLAGS = -version-number $(subst .,:,$(PACKAGE_VERSION))
 if CXXLINK_REQUIRED
 libadmsslutil_la_LINK = $(CXXLINK)
--- adminutil-1.1.0/adminutil.pc.in.orig	2007-04-04 21:37:35.000000000 +0200
+++ adminutil-1.1.0/adminutil.pc.in	2007-06-16 11:00:35.900667997 +0200
@@ -6,6 +6,6 @@
 Name: @PACKAGE_NAME@
 Description: Utility library for directory server administration
 Version: @PACKAGE_VERSION@
-Requires: nspr, nss, svrcore, mozldap, icu
+Requires: nspr, nss, mozldap, icu
 Libs: -ladmsslutil -ladminutil
 Cflags: -I${includedir}/admsslutil -I${includedir}/adminutil
